susemanager-utils/testing/docker/scripts/reportdb_schema_idempotency_test_pgsql.py

```diff
@@ -227,6 +229,11 @@ def dump_database(dump_name, excluded_tables=None):
     else:
         # pylint: disable-next=consider-using-f-string
         raise RuntimeError("Could not dump %s!" % db_name)
+    cleanup_cmd = f"/usr/bin/sed -i 's/^\(\\\\u\?n\?restrict \).*$/\\1/g' {dump_name}"
+    if run_command(cleanup_cmd):
+        print(f"{dump_name} cleanuped")
```

Comment:
[nit] `cleanuped` is grammatically incorrect, but more importantly, perhaps we can be more specific:

```suggestion
        print(f"Cleaned unrestrict/restrict tokens from {dump_name} ")
```